### ISSUE TITLE
update/simplify support for fba_plot_results

### ISSUE THREAD

**Issue by sbailey** [desihub/fiberassign#332]
fba_plot_results currently requires a few extra steps to work on gzipped fiberassign files, plus combining with information from tiles files in other locations.  Instructions from an email from Ted:

```
source /global/common/software/desi/desi_environment.sh master

# Make a working directory
cd work
gunzip -c /global/cfs/cdirs/desi/survey/fiberassign/SV3/20210413/fiberassign-000413.fits.gz > fiberassign-000413.fits

# Make a PDF plot
fba_plot_results --prefix 'fiberassign-' --dir . --real_shapes --footprint /global/cfs/cdirs/desi/survey/fiberassign/SV3/20210413/000413-tiles.fits --rundate "2021-04-13T20:00:00"
```

For consideration:
  * Personally I would find it more convenient if there was an option for fba_plot_results to work on a single input file given as an explicit input rather than the combination of --prefix and --dir . with it making assumptions about extentions (gzipped or not...)
  * What information does it get from the tiles file, and is that information available in the header of the fiberassign file anyway (or should it?)
  * Ditto for rundate: isn't that in the header of the file?

i.e. could this be simplified to `fba_plot_results -i fiberassign-NNNNNN.fits.gz -o fiberassign-NNNNNN.pdf' ?

On one hand this is lower priority than the tickets about actual fiberassign functionality, but this is a really useful tool for deep debugging once you know how to run it.

**Comment by tskisner** [comment]
Work in progress in #336.  Basically the new syntax there is:
```
fba_plot --real_shapes file1.fits file2.fits file3.fits ...
```
With generated PDFs named after the root name of each file.

**Comment by sbailey** [comment]
implemented in PR #336; closing.
